lib/view/students/students_screen.dart

```diff
@@ -64,17 +64,25 @@ class StudentsPage extends StatelessWidget {
         controllerOfDescription: TextEditingController(text: ""),
         birthdate: "Select Date",
         controllerOfHours: TextEditingController(text: ""));
-        // -------------------------------------->
+    // -------------------------------------->
```

Comment:
This comment/divider is not necessary.
I do not understand the issue here.

Reply:
solved